### PR TITLE
#5 Created Project Skelleton

### PR DESCRIPTION
**Pull request by DenysMelnychenko**
connected to #5

### CODE REVIEW

**Review by kortov — CHANGES_REQUESTED** [review]
We should pull requests into our own branches (e.g. JujaLabs:denysmelnychenko ), not in JujaLabs:master

**Comment by kortov** [comment]
Code looks good

**Comment by DenysMelnychenko** [comment]
Sorry, it was already late and I did not see that the master branch was selected

**Review by kortov — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>juja.sqlcmd</groupId>
+    <artifactId>sqlcmd</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>sqlcmd</name>
+    <url>http://maven.apache.org</url>
```
Comment (Shtramak) [inline pom.xml:11]: "url" is unnecessary tag that represents URL where the project lives. In this case, for example, you could use "https://github.com/DenysMelnychenko/sqlcmd"

File: workflow.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# Github and Team workflow for JuJa SQLcmd
```
Comment (Shtramak) [inline workflow.md:1]: I don't see any reason that file to be in a project folder, it doesn't relate to a project. Recommend to exclude it from remote repository. To remove file from index, but keep it on HDD you should use git rm --cached

**Review by Shtramak — APPROVED** [review]
Pay your attention on commits naming, they should contain a number of task. For examle the name of your last commit should be "#5: Created Project Skelleton". With "#5" you can go directly to a task even from your github repo. If you'd like to rename the last commit, you can use git commit --amend -m "#5: Created Project Skelleton". I had the same issue

File: workflow.md (hunk 2 above)
Comment (DenysMelnychenko) [reply workflow.md:1]: It was already added to the project structure. That why I didn't exclude it from repo.

**Review by ivshapovalov — APPROVED** [review]

**Review by adaleorion — APPROVED** [review]
thanks, nice job 👍

**Review by AVM82 — APPROVED** [review]
Looks good to me

**Pull request merged**